File: js/half.js
function half(number, strings) {
    for (var i = 0; i < strings.length; i++) {
        if (strings[i].length % 2 == 0) {
            string = strings[i]
            console.log(string.substring(0, string.length/2))
        }
    }
}
strings = ['pierwszy', 'lubiec', 'ktotozrobi']
console.log(half(3, strings))

File: c#/half.cs
using System;

namespace ConsoleApp25
{
    class Program
    {
        static void half(int number, string[] strings)
        {
            for (int i = 0; i< number; i++)
            {
                if (strings[i].Length % 2 == 0)
                {
                    string text = strings[i];
                    Console.WriteLine(text.Substring(0, text.Length/2));
                }
            }
        }
        static void Main(string[] args)
        {
            string[] strings = { "pierwszy", "lubiec", "ktotozrobi" };
            half(3, strings);
        }
    }
}
